fili-core/src/main/java/com/yahoo/bard/webservice/data/config/dimension/LookupDefaultDimensionConfig.java

```diff
@@ -0,0 +1,102 @@
+
+/**
+ * A Lookup Dimension instance holds all of the information needed to construct a Lookup Dimension.
+ */
+public class LookupDefaultDimensionConfig extends DefaultDimensionConfig implements LookupDimensionConfig {
+  private final List<String> namespaces;
+
+  /**
+   * Construct a LookupDefaultDimensionConfig Instance from Dimension Name , dimension fields and
+   * default dimension fields.
+   *
+   * @param apiName  The API Name is the external, end-user-facing name for the dimension.
+   * @param physicalName  The internal, physical name for the dimension.
+   * @param description  A description of the dimension and its meaning.
+   * @param longName  The Long Name is the external, end-user-facing long  name for the dimension.
+   * @param category  The Category is the external, end-user-facing category for the dimension.
+   * @param fields  The set of fields for this dimension.
+   * @param defaultDimensionFields  The default set of fields for this dimension to be shown in the response.
+   * @param keyValueStore  The key value store holding dimension row data.
+   * @param searchProvider  The search provider for field value lookups on this dimension.
+   * @param namespaces  A list of namespaces used to configure the Lookup dimension.
+   */
+  public LookupDefaultDimensionConfig(
+      @NotNull DimensionName apiName,
+      @NotNull String physicalName,
+      @NotNull String description,
+      @NotNull String longName,
+      @NotNull String category,
+      @NotNull LinkedHashSet<DimensionField> fields,
+      @NotNull LinkedHashSet<DimensionField> defaultDimensionFields,
+      @NotNull KeyValueStore keyValueStore,
+      @NotNull SearchProvider searchProvider,
+      @NotNull List<String> namespaces
+  ) {
+    super (
+        apiName,
+        physicalName,
+        description,
+        longName,
+        category,
+        fields,
+        defaultDimensionFields,
+        keyValueStore,
+        searchProvider
+    );
+    this.namespaces = namespaces;
+  }
+
+  /**
+   * Construct a LookupDefaultDimensionConfig Instance from Dimension Name , dimension fields and  default dimension fields.
+   *
+   * @param apiName  The API Name is the external, end-user-facing name for the dimension.
+   * @param physicalName  The internal, physical name for the dimension.
+   * @param description  A description of the dimension and its meaning.
+   * @param longName  The Long Name is the external, end-user-facing long  name for the dimension.
+   * @param category  The Category is the external, end-user-facing category for the dimension.
+   * @param defaultDimensionFields  The default set of fields for this dimension to be shown in the response.
+   * @param keyValueStore  The key value store holding dimension row data.
+   * @param searchProvider  The search provider for field value lookups on this dimension.
+   * @param namespaces  A list of namespaces used to configure the Lookup dimension.
+   */
```

Comment:
These javadocs need to call out the differences between these two constructors. It's not obvious what the differences are at a quick glance.